Cognigy/WebchatWidget
# Feature/16332 rating functionality

This PR adds a rating functionality to our webchat.

To Test:
1. Deploy the webchat with the setting "enableRating" set to "always" or "once" connected to your local cognigy deployment
2. Using the webchat, you will see the button to give a rating in the Header (it will disappear after giving a rating if the setting is set to "once", please test that as well).
3. Using the rating dialog, you can send a rating message. It should also be displayed in the message history.
4. To test the message coming in, you can have a quick look at your logs. 
Use Odata to verify (exchange YOURAGENTPROJECTID and YOURAPIKEY with your values): `http://odata/v2.1/Conversations?$filter=projectId%20eq%20%27YOURAGENTPROJECTID%27&apikey=YOURAPIKEY`

If already merged and working for you, you can also e2e review this using the newest insight tools.
